client/src/components/clan/ForumEntry.jsx

```diff
@@ -9,24 +11,57 @@ import { Switch, Route } from 'react-router-dom';
 class ForumEntry extends React.Component {
   constructor (props) {
     super(props);
+    this.handleDownVote.bind(this);
+    this.handleUpVote.bind(this);
+    this.state = {
+      votes: 0
+    };
   }
 
-  render() {
+  handleUpVote (e) {
+    console.log('Upvoted');
+    let votes = this.state.votes;
+    this.setState({
+      votes: votes++
```

Comment:
We may want to let the express layer handle this logic. If a user wants to up or downvote, that could be specified in the request body to the endpoint of the post. If we duplicate the logic on the client, it would also keep track of whether the signed-in user has yet upvoted and downvoted.